bigbluebutton-html5/imports/ui/components/end-meeting-confirmation/service.js

```diff
@@ -0,0 +1,34 @@
+import Users from '/imports/api/users';
+import Auth from '/imports/ui/services/auth';
+import Meetings from '/imports/api/meetings';
+
+const userFindSorting = {
+  emojiTime: 1,
+  role: 1,
+  phoneUser: 1,
+  sortName: 1,
+  userId: 1,
+};
+
+const getMeetingTitle = () => {
+  const meeting = Meetings.findOne({
+    meetingId: Auth.meetingID,
+  }, { fields: { 'meetingProp.name': 1, 'breakoutProps.sequence': 1 } });
+
+  return meeting.meetingProp.name;
+};
+
+const getUsers = () => {
```

Comment:
Given that we only need to know `users.length` it would be more efficient to find(..).count() and with the `fields` option set (afaik we needed to pass _id: 1 or something like it)
No need of sorting or fetching